WARNING:root:Mensaje a nivel warning
INFO:root:Mensaje a nivel info
DEBUG:root:Mensaje a nivel debug
ERROR:root:Ocurrio un error en la base de datos
DEBUG:root:Se realizo la conexion con exito
2021-05-26 15:07:46,967
2021-05-26 15:07:46,968
2021-05-26 15:07:46,970
2021-05-26 15:07:46,970
2021-05-26 15:07:46,971
2021-05-26 15:09:45,722: WARNING
2021-05-26 15:09:45,723: INFO
2021-05-26 15:09:45,723: DEBUG
2021-05-26 15:09:45,724: ERROR
2021-05-26 15:09:45,724: DEBUG
2021-05-26 15:20:13,498: WARNING [logger_base.py]
2021-05-26 15:20:13,512: INFO [logger_base.py]
2021-05-26 15:20:13,513: DEBUG [logger_base.py]
2021-05-26 15:20:13,513: ERROR [logger_base.py]
2021-05-26 15:20:13,514: DEBUG [logger_base.py]
2021-05-26 15:21:09,844: WARNING [logger_base.py:12]
2021-05-26 15:21:09,845: INFO [logger_base.py:13]
2021-05-26 15:21:09,846: DEBUG [logger_base.py:14]
2021-05-26 15:21:09,847: ERROR [logger_base.py:15]
2021-05-26 15:21:09,847: DEBUG [logger_base.py:16]
2021-05-26 15:22:04,378: WARNING [logger_base.py:12 Mensaje a nivel warning]
2021-05-26 15:22:04,378: INFO [logger_base.py:13 Mensaje a nivel info]
2021-05-26 15:22:04,378: DEBUG [logger_base.py:14 Mensaje a nivel debug]
2021-05-26 15:22:04,379: ERROR [logger_base.py:15 Ocurrio un error en la base de datos]
2021-05-26 15:22:04,379: DEBUG [logger_base.py:16 Se realizo la conexion con exito]
03:23:55 PM: WARNING [logger_base.py:13 Mensaje a nivel warning]
03:23:55 PM: INFO [logger_base.py:14 Mensaje a nivel info]
03:23:55 PM: DEBUG [logger_base.py:15 Mensaje a nivel debug]
03:23:55 PM: ERROR [logger_base.py:16 Ocurrio un error en la base de datos]
03:23:55 PM: DEBUG [logger_base.py:17 Se realizo la conexion con exito]
03:28:43 PM: INFO [conexion.py:3 Desde conexion]
03:48:01 PM: DEBUG [conexion.py:23 Conexion exitosa: <connection object at 0x00000274820E1E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
03:48:01 PM: INFO [conexion.py:32 <connection object at 0x00000274820E1E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
03:49:25 PM: ERROR [conexion.py:26 Error al conectar a la BD: FATAL:  database "test_db1" does not exist]
01:49:19 PM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x0000021BFC8ECE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
01:49:19 PM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x0000021BFE446580; closed: 0>]
01:49:19 PM: INFO [conexion.py:59 <cursor object at 0x0000021BFE446580; closed: 0>]
01:49:19 PM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
02:56:27 PM: DEBUG [persona.py:20 ID Persona: 1, Nombre: Juan, Apellido: Perez, Email: jperez@gmail.com]
02:57:16 PM: DEBUG [persona.py:20 
            f'ID Persona: {self.__id_persona}, '
            f'Nombre: {self.__nombre}, '
            f'Apellido: {self.__apellido}, '
            f'Email: {self.__email}'
        ]
02:58:41 PM: DEBUG [persona.py:20 ID Persona: 1, 
Nombre: Juan, 
Apellido: Perez, 
Email: jperez@gmail.com]
02:59:00 PM: DEBUG [persona.py:20 
ID Persona: 1, 
Nombre: Juan, 
Apellido: Perez, 
Email: jperez@gmail.com]
03:01:20 PM: DEBUG [persona.py:20 
ID Persona: 1, 
Nombre: Juan, 
Apellido: Perez, 
Email: jperez@gmail.com]
03:05:12 PM: DEBUG [persona.py:20 ID Persona: 1, Nombre: Juan, Apellido: Perez, Email: jperez@gmail.com]
03:05:12 PM: DEBUG [persona.py:23 ID Persona: None, Nombre: Manolo, Apellido: Lama, Email: melama@gmail.com]
03:07:44 PM: DEBUG [persona.py:20 ID Persona: 1, Nombre: Juan, Apellido: Perez, Email: jperez@gmail.com]
03:07:44 PM: DEBUG [persona.py:23 ID Persona: None, Nombre: Manolo, Apellido: Lama, Email: melama@gmail.com]
03:07:44 PM: DEBUG [persona.py:26 ID Persona: None, Nombre: Paco, Apellido: Gonzales, Email: pacogo@gmail.com]
07:13:57 PM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x0000020127E46E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
07:13:57 PM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x0000020129AC54A0; closed: 0>]
07:13:57 PM: DEBUG [persona_dao.py:13 b'SELECT * FROM persona ORDER BY id_persona']
07:13:57 PM: DEBUG [persona_dao.py:25 ID Persona: 2, Nombre: Perro, Apellido: Viejo, Email: perrovieho@gmail.com]
07:13:57 PM: DEBUG [persona_dao.py:25 ID Persona: 3, Nombre: Oze, Apellido: Migué, Email: ozemigue@mail.com]
07:13:57 PM: DEBUG [persona_dao.py:25 ID Persona: 9, Nombre: Wanillo, Apellido: Kokunero, Email: wanillokokunero@gmail.com]
07:13:57 PM: DEBUG [persona_dao.py:25 ID Persona: 14, Nombre: Paco, Apellido: Pa, Email: pacopa@gmail.com]
07:13:57 PM: DEBUG [persona_dao.py:25 ID Persona: 15, Nombre: Kike, Apellido: Montilla, Email: kikemontilla@gmail.com]
07:13:57 PM: DEBUG [persona_dao.py:25 ID Persona: 17, Nombre: Aitor, Apellido: Menta, Email: aitormenta@gmail.com]
07:34:50 PM: DEBUG [persona.py:20 
            ID Persona: 1, '
            Nombre: Juan, '
            Apellido: Perez, '
            Email: jperez@gmail.com'
        ]
07:34:50 PM: DEBUG [persona.py:23 
            ID Persona: None, '
            Nombre: Manolo, '
            Apellido: Lama, '
            Email: melama@gmail.com'
        ]
07:34:50 PM: DEBUG [persona.py:27 
            ID Persona: None, '
            Nombre: Paco, '
            Apellido: Gonzales, '
            Email: pacogo@gmail.com'
        ]
08:46:30 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000001DE9223CE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
08:46:30 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000001DE93D76580; closed: 0>]
08:46:30 AM: INFO [conexion.py:59 <cursor object at 0x000001DE93D76580; closed: 0>]
08:46:30 AM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
08:52:01 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000002DDE8BE6E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
08:52:01 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000002DDEA737580; closed: 0>]
08:52:01 AM: DEBUG [persona_dao.py:13 b'SELECT * FROM persona ORDER BY id_persona']
08:53:13 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x0000020E39EB6E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
08:53:13 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x0000020E3BA47580; closed: 0>]
08:53:13 AM: DEBUG [persona_dao.py:13 b'SELECT * FROM persona ORDER BY id_persona']
08:53:13 AM: DEBUG [persona_dao.py:25 ID Persona: 2, Nombre: Perro, Apellido: Viejo, Email: perrovieho@gmail.com]
08:53:13 AM: DEBUG [persona_dao.py:25 ID Persona: 3, Nombre: Oze, Apellido: Migué, Email: ozemigue@mail.com]
08:53:13 AM: DEBUG [persona_dao.py:25 ID Persona: 9, Nombre: Wanillo, Apellido: Kokunero, Email: wanillokokunero@gmail.com]
08:53:13 AM: DEBUG [persona_dao.py:25 ID Persona: 14, Nombre: Paco, Apellido: Pa, Email: pacopa@gmail.com]
08:53:13 AM: DEBUG [persona_dao.py:25 ID Persona: 15, Nombre: Kike, Apellido: Montilla, Email: kikemontilla@gmail.com]
08:53:13 AM: DEBUG [persona_dao.py:25 ID Persona: 17, Nombre: Aitor, Apellido: Menta, Email: aitormenta@gmail.com]
08:53:37 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x00000181874F6E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
08:53:37 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x0000018189177580; closed: 0>]
08:53:37 AM: DEBUG [persona_dao.py:13 b'SELECT * FROM persona ORDER BY id_persona']
09:15:30 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x00000153E0C8BE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
09:15:30 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x00000153E27FA580; closed: 0>]
09:15:30 AM: DEBUG [persona_dao.py:29 b'INSERT INTO persona(nombre,apellido,email) VALUES(%s,%s,%s)']
09:15:30 AM: DEBUG [persona_dao.py:30 Persona a insertar: ID Persona: None, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
09:15:30 AM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
09:15:30 AM: DEBUG [persona_dao.py:47 Registros insertados: 1]
09:18:40 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x0000024839A0BE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
09:18:40 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000002483B56A580; closed: 0>]
09:18:40 AM: DEBUG [persona_dao.py:29 b'INSERT INTO persona(nombre,apellido,email) VALUES(%s,%s,%s)']
09:18:40 AM: DEBUG [persona_dao.py:30 Persona a insertar: ID Persona: None, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
09:18:40 AM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
09:18:40 AM: DEBUG [persona_dao.py:44 Registros insertados: 1]
09:18:40 AM: DEBUG [persona_dao.py:14 b'SELECT * FROM persona ORDER BY id_persona']
09:19:00 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000002755F32BE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
09:19:00 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x0000027560E7A580; closed: 0>]
09:19:00 AM: DEBUG [persona_dao.py:29 b'INSERT INTO persona(nombre,apellido,email) VALUES(%s,%s,%s)']
09:19:00 AM: DEBUG [persona_dao.py:30 Persona a insertar: ID Persona: None, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
09:19:00 AM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
09:19:00 AM: DEBUG [persona_dao.py:44 Registros insertados: 1]
09:19:00 AM: DEBUG [persona_dao.py:14 b'SELECT * FROM persona ORDER BY id_persona']
09:20:03 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000002B42F01BE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
09:20:03 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000002B430B7B580; closed: 0>]
09:20:03 AM: DEBUG [persona_dao.py:14 b'SELECT * FROM persona ORDER BY id_persona']
09:20:03 AM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
09:20:03 AM: DEBUG [persona_dao.py:48 ID Persona: 2, Nombre: Perro, Apellido: Viejo, Email: perrovieho@gmail.com]
09:20:03 AM: DEBUG [persona_dao.py:48 ID Persona: 3, Nombre: Oze, Apellido: Migué, Email: ozemigue@mail.com]
09:20:03 AM: DEBUG [persona_dao.py:48 ID Persona: 9, Nombre: Wanillo, Apellido: Kokunero, Email: wanillokokunero@gmail.com]
09:20:03 AM: DEBUG [persona_dao.py:48 ID Persona: 14, Nombre: Paco, Apellido: Pa, Email: pacopa@gmail.com]
09:20:03 AM: DEBUG [persona_dao.py:48 ID Persona: 15, Nombre: Kike, Apellido: Montilla, Email: kikemontilla@gmail.com]
09:20:03 AM: DEBUG [persona_dao.py:48 ID Persona: 17, Nombre: Aitor, Apellido: Menta, Email: aitormenta@gmail.com]
09:20:03 AM: DEBUG [persona_dao.py:48 ID Persona: 23, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
09:20:03 AM: DEBUG [persona_dao.py:48 ID Persona: 24, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
09:20:03 AM: DEBUG [persona_dao.py:48 ID Persona: 25, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
09:20:35 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000001B0498FBE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
09:20:35 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000001B04B46A580; closed: 0>]
09:20:35 AM: DEBUG [persona_dao.py:29 b'INSERT INTO persona(nombre,apellido,email) VALUES(%s,%s,%s)']
09:20:35 AM: DEBUG [persona_dao.py:30 Persona a insertar: ID Persona: None, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
09:20:35 AM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
09:20:35 AM: DEBUG [persona_dao.py:44 Registros insertados: 1]
09:20:35 AM: DEBUG [persona_dao.py:14 b'SELECT * FROM persona ORDER BY id_persona']
09:21:33 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000001B4A775BE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
09:21:33 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000001B4A92AA580; closed: 0>]
09:21:33 AM: DEBUG [persona_dao.py:29 b'INSERT INTO persona(nombre,apellido,email) VALUES(%s,%s,%s)']
09:21:33 AM: DEBUG [persona_dao.py:30 Persona a insertar: ID Persona: None, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
09:21:33 AM: DEBUG [persona_dao.py:44 Registros insertados: 1]
09:21:33 AM: DEBUG [persona_dao.py:14 b'SELECT * FROM persona ORDER BY id_persona']
09:21:33 AM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
09:21:33 AM: DEBUG [persona_dao.py:48 ID Persona: 2, Nombre: Perro, Apellido: Viejo, Email: perrovieho@gmail.com]
09:21:33 AM: DEBUG [persona_dao.py:48 ID Persona: 3, Nombre: Oze, Apellido: Migué, Email: ozemigue@mail.com]
09:21:33 AM: DEBUG [persona_dao.py:48 ID Persona: 9, Nombre: Wanillo, Apellido: Kokunero, Email: wanillokokunero@gmail.com]
09:21:33 AM: DEBUG [persona_dao.py:48 ID Persona: 14, Nombre: Paco, Apellido: Pa, Email: pacopa@gmail.com]
09:21:33 AM: DEBUG [persona_dao.py:48 ID Persona: 15, Nombre: Kike, Apellido: Montilla, Email: kikemontilla@gmail.com]
09:21:33 AM: DEBUG [persona_dao.py:48 ID Persona: 17, Nombre: Aitor, Apellido: Menta, Email: aitormenta@gmail.com]
09:21:33 AM: DEBUG [persona_dao.py:48 ID Persona: 23, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
09:21:33 AM: DEBUG [persona_dao.py:48 ID Persona: 24, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
09:21:33 AM: DEBUG [persona_dao.py:48 ID Persona: 25, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
09:21:33 AM: DEBUG [persona_dao.py:48 ID Persona: 26, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
09:21:33 AM: DEBUG [persona_dao.py:48 ID Persona: 27, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
09:41:50 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x00000211F51CDE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
09:41:50 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x00000211F6D1A580; closed: 0>]
09:41:50 AM: DEBUG [persona_dao.py:47 b'UPDATE persona SET nombre = %s, apellido = %s, email = %s WHERE id_persona = %s']
09:41:50 AM: DEBUG [persona_dao.py:48 Persona a actualizar: ID Persona: None, Nombre: Monica, Apellido: Fernandez, Email: mfernandez@gmail.com]
09:41:52 AM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
09:41:52 AM: DEBUG [persona_dao.py:71 Personas actualizadas: 1]
09:49:13 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x00000260D3EEEE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
09:49:13 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x00000260D5A5C580; closed: 0>]
09:49:13 AM: DEBUG [persona_dao.py:48 b'UPDATE persona SET nombre = %s, apellido = %s, email = %s WHERE id_persona = %s']
09:49:13 AM: DEBUG [persona_dao.py:49 Datos a actualizar: ID Persona: None, Nombre: Monica, Apellido: Fernandez, Email: mfernandez@gmail.com]
09:49:45 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000001E6FE33EE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
09:49:45 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000001E6FFECC580; closed: 0>]
09:49:45 AM: DEBUG [persona_dao.py:48 b'UPDATE persona SET nombre = %s, apellido = %s, email = %s WHERE id_persona = %s']
09:49:45 AM: DEBUG [persona_dao.py:49 Nueva persona: ID Persona: None, Nombre: Monica, Apellido: Fernandez, Email: mfernandez@gmail.com]
09:49:56 AM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
09:49:56 AM: DEBUG [persona_dao.py:83 Personas actualizadas: 1]
09:59:20 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x0000021C19A89E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
09:59:20 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x0000021C1B5EB580; closed: 0>]
09:59:20 AM: DEBUG [persona_dao.py:48 b'UPDATE persona SET nombre = %s, apellido = %s, email = %s WHERE id_persona = %s']
09:59:20 AM: DEBUG [persona_dao.py:49 Nueva persona: ID Persona: 2, Nombre: Monica, Apellido: Fernandez, Email: mfernandez@gmail.com]
09:59:20 AM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
09:59:20 AM: DEBUG [persona_dao.py:86 Personas actualizadas: 1]
09:59:25 AM: DEBUG [persona_dao.py:65 b'DELETE FROM persona WHERE id_persona = %s']
09:59:25 AM: DEBUG [persona_dao.py:66 Persona a eliminar: ID Persona: 23, Nombre: None, Apellido: None, Email: None]
09:59:46 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000001E28A9CEE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
09:59:46 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000001E28C54C580; closed: 0>]
09:59:46 AM: DEBUG [persona_dao.py:65 b'DELETE FROM persona WHERE id_persona = %s']
09:59:46 AM: DEBUG [persona_dao.py:66 Persona a eliminar: ID Persona: 23, Nombre: None, Apellido: None, Email: None]
09:59:46 AM: ERROR [persona_dao.py:73 Error al eliminar: 'Persona' object has no attribute 'getI_ID']
09:59:46 AM: DEBUG [persona_dao.py:90 Personas eliminadas: None]
10:00:28 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000002AB3D9AEE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:00:28 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000002AB3F52B580; closed: 0>]
10:00:28 AM: DEBUG [persona_dao.py:65 b'DELETE FROM persona WHERE id_persona = %s']
10:00:28 AM: DEBUG [persona_dao.py:66 Persona a eliminar: ID Persona: 23, Nombre: None, Apellido: None, Email: None]
10:00:28 AM: ERROR [persona_dao.py:73 Error al eliminar: 'int' object does not support indexing]
10:00:28 AM: DEBUG [persona_dao.py:90 Personas eliminadas: None]
10:01:34 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000002260EFFEE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:01:34 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x0000022610B6B580; closed: 0>]
10:01:34 AM: DEBUG [persona_dao.py:65 b'DELETE FROM persona WHERE id_persona = %s']
10:01:34 AM: DEBUG [persona_dao.py:66 Persona a eliminar: ID Persona: 23, Nombre: None, Apellido: None, Email: None]
10:01:34 AM: ERROR [persona_dao.py:73 Error al eliminar: not all arguments converted during string formatting]
10:01:34 AM: DEBUG [persona_dao.py:90 Personas eliminadas: None]
10:02:02 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x00000216214EEE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:02:02 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000002162305B580; closed: 0>]
10:02:02 AM: DEBUG [persona_dao.py:65 b'DELETE FROM persona WHERE id_persona = %s']
10:02:02 AM: DEBUG [persona_dao.py:66 Persona a eliminar: ID Persona: 23, Nombre: None, Apellido: None, Email: None]
10:02:02 AM: ERROR [persona_dao.py:73 Error al eliminar: 'int' object does not support indexing]
10:02:02 AM: DEBUG [persona_dao.py:90 Personas eliminadas: None]
10:04:16 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000001DF7109EE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:04:16 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000001DF72BEB580; closed: 0>]
10:04:16 AM: DEBUG [persona_dao.py:65 b'DELETE FROM persona WHERE id_persona = %s']
10:04:16 AM: DEBUG [persona_dao.py:66 Persona a eliminar: ID Persona: ['2', '3'], Nombre: None, Apellido: None, Email: None]
10:04:16 AM: ERROR [persona_dao.py:73 Error al eliminar: not all arguments converted during string formatting]
10:04:16 AM: DEBUG [persona_dao.py:90 Personas eliminadas: None]
10:05:07 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x00000279E617EE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:05:07 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x00000279E7CFB580; closed: 0>]
10:05:07 AM: DEBUG [persona_dao.py:65 b'DELETE FROM persona WHERE id_persona = %s']
10:05:07 AM: DEBUG [persona_dao.py:66 Persona a eliminar: ID Persona: list['23'], Nombre: None, Apellido: None, Email: None]
10:05:07 AM: ERROR [persona_dao.py:73 Error al eliminar: types.GenericAlias is not a sequence]
10:05:07 AM: DEBUG [persona_dao.py:90 Personas eliminadas: None]
10:05:48 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000001978F90EE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:05:48 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000001979148B580; closed: 0>]
10:05:48 AM: DEBUG [persona_dao.py:65 b'DELETE FROM persona WHERE id_persona = %s']
10:05:48 AM: DEBUG [persona_dao.py:66 Persona a eliminar: ID Persona: list[23], Nombre: None, Apellido: None, Email: None]
10:05:48 AM: ERROR [persona_dao.py:73 Error al eliminar: types.GenericAlias is not a sequence]
10:05:48 AM: DEBUG [persona_dao.py:90 Personas eliminadas: None]
10:06:35 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000002430EAEEE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:06:35 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000002431062B580; closed: 0>]
10:06:35 AM: DEBUG [persona_dao.py:65 b'DELETE FROM persona WHERE id_persona = %s']
10:06:35 AM: DEBUG [persona_dao.py:66 Persona a eliminar: ID Persona: 23, Nombre: None, Apellido: None, Email: None]
10:06:35 AM: ERROR [persona_dao.py:73 Error al eliminar: 'int' object does not support indexing]
10:06:35 AM: DEBUG [persona_dao.py:90 Personas eliminadas: None]
10:07:02 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x0000022C154CEE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:07:02 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x0000022C1701B580; closed: 0>]
10:07:02 AM: DEBUG [persona_dao.py:65 b'DELETE FROM persona WHERE id_persona = %s']
10:07:02 AM: DEBUG [persona_dao.py:66 Persona a eliminar: ID Persona: 23, Nombre: None, Apellido: None, Email: None]
10:07:02 AM: ERROR [persona_dao.py:73 Error al eliminar: 'int' object does not support indexing]
10:07:02 AM: DEBUG [persona_dao.py:90 Personas eliminadas: None]
10:07:46 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000001F2DA00EE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:07:46 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000001F2DBB8B580; closed: 0>]
10:07:46 AM: DEBUG [persona_dao.py:65 b'DELETE FROM persona WHERE id_persona = %s']
10:07:46 AM: DEBUG [persona_dao.py:66 Persona a eliminar: ID Persona: 23, Nombre: None, Apellido: None, Email: None]
10:07:46 AM: ERROR [persona_dao.py:73 Error al eliminar: 'psycopg2.extensions.cursor' object has no attribute 'commit']
10:07:46 AM: DEBUG [persona_dao.py:90 Personas eliminadas: None]
10:08:12 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000002AD5464EE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:08:12 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000002AD561BB580; closed: 0>]
10:08:12 AM: DEBUG [persona_dao.py:65 b'DELETE FROM persona WHERE id_persona = %s']
10:08:12 AM: DEBUG [persona_dao.py:66 Persona a eliminar: ID Persona: 23, Nombre: None, Apellido: None, Email: None]
10:08:12 AM: DEBUG [persona_dao.py:90 Personas eliminadas: 1]
10:09:20 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000001A3E52EEE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:09:20 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000001A3E6E5C580; closed: 0>]
10:09:20 AM: DEBUG [persona_dao.py:31 b'INSERT INTO persona(nombre,apellido,email) VALUES(%s,%s,%s)']
10:09:20 AM: DEBUG [persona_dao.py:32 Persona a insertar: ID Persona: None, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
10:09:20 AM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
10:09:20 AM: DEBUG [persona_dao.py:82 Registros insertados: 1]
10:09:40 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000001A24F82EE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:09:40 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000001A25138B580; closed: 0>]
10:09:40 AM: DEBUG [persona_dao.py:65 b'DELETE FROM persona WHERE id_persona = %s']
10:09:40 AM: DEBUG [persona_dao.py:66 Persona a eliminar: ID Persona: 28, Nombre: None, Apellido: None, Email: None]
10:09:40 AM: DEBUG [persona_dao.py:90 Personas eliminadas: 1]
10:20:12 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x00000281B685EE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:20:12 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x00000281B838B660; closed: 0>]
10:20:12 AM: DEBUG [persona_dao.py:31 b'INSERT INTO persona(nombre,apellido,email) VALUES(%s,%s,%s)']
10:20:12 AM: DEBUG [persona_dao.py:32 Persona a insertar: ID Persona: None, Nombre: Adrian, Apellido: Carmona, Email: acarmona@gmail.com]
10:20:12 AM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
10:20:12 AM: DEBUG [persona_dao.py:84 Registros insertados: 1]
10:20:27 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000001DD8264EE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:20:27 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000001DD841CB660; closed: 0>]
10:20:27 AM: DEBUG [persona_dao.py:65 b'DELETE FROM persona WHERE id_persona = %s']
10:20:27 AM: DEBUG [persona_dao.py:66 Persona a eliminar: ID Persona: 29, Nombre: None, Apellido: None, Email: None]
10:20:27 AM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
10:20:27 AM: DEBUG [persona_dao.py:92 Personas eliminadas: 1]
01:52:28 PM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x0000020DBD5CCE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
01:52:28 PM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x0000020DBF11C660; closed: 0>]
01:52:28 PM: DEBUG [persona_dao.py:16 b'SELECT * FROM persona ORDER BY id_persona']
01:52:28 PM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
01:52:59 PM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000001C536FDDE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
01:52:59 PM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000001C538B2C660; closed: 0>]
01:52:59 PM: DEBUG [persona_dao.py:16 b'SELECT * FROM persona ORDER BY id_persona']
01:52:59 PM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
01:53:33 PM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x000001F31DCBDE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
01:53:33 PM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x000001F31F82B660; closed: 0>]
01:53:33 PM: DEBUG [persona_dao.py:16 b'SELECT * FROM persona ORDER BY id_persona']
01:53:33 PM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
01:53:53 PM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x0000024AA4B6EE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
01:53:53 PM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x0000024AA66DB660; closed: 0>]
01:53:53 PM: DEBUG [persona_dao.py:16 b'SELECT * FROM persona ORDER BY id_persona']
01:53:53 PM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
01:53:53 PM: DEBUG [persona_dao.py:80 ID Persona: 2, Nombre: Monica, Apellido: Fernandez, Email: mfernandez@gmail.com]
01:53:53 PM: DEBUG [persona_dao.py:80 ID Persona: 3, Nombre: Oze, Apellido: Migué, Email: ozemigue@mail.com]
01:53:53 PM: DEBUG [persona_dao.py:80 ID Persona: 9, Nombre: Wanillo, Apellido: Kokunero, Email: wanillokokunero@gmail.com]
01:53:53 PM: DEBUG [persona_dao.py:80 ID Persona: 14, Nombre: Paco, Apellido: Pa, Email: pacopa@gmail.com]
01:53:53 PM: DEBUG [persona_dao.py:80 ID Persona: 15, Nombre: Kike, Apellido: Montilla, Email: kikemontilla@gmail.com]
01:53:53 PM: DEBUG [persona_dao.py:80 ID Persona: 17, Nombre: Aitor, Apellido: Menta, Email: aitormenta@gmail.com]
01:54:49 PM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x00000136B046DE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
01:54:49 PM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x00000136B1FDC660; closed: 0>]
01:54:49 PM: DEBUG [persona_dao.py:16 b'SELECT * FROM persona ORDER BY id_persona']
01:54:49 PM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
01:54:49 PM: DEBUG [persona_dao.py:94 [<persona.Persona object at 0x00000136B1FD26D0>, <persona.Persona object at 0x00000136B1FD2640>, <persona.Persona object at 0x00000136B1FD27C0>, <persona.Persona object at 0x00000136B1FD25E0>, <persona.Persona object at 0x00000136B1FD2520>, <persona.Persona object at 0x00000136B1FD24C0>]]
10:13:45 PM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x00000272B8204E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
10:13:45 PM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x00000272B9D66660; closed: 0>]
10:13:45 PM: DEBUG [persona_dao.py:16 b'SELECT * FROM persona ORDER BY id_persona']
10:13:45 PM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
10:13:45 PM: DEBUG [persona_dao.py:80 ID Persona: 2, Nombre: Monica, Apellido: Fernandez, Email: mfernandez@gmail.com]
10:13:45 PM: DEBUG [persona_dao.py:80 ID Persona: 3, Nombre: Oze, Apellido: Migué, Email: ozemigue@mail.com]
10:13:45 PM: DEBUG [persona_dao.py:80 ID Persona: 9, Nombre: Wanillo, Apellido: Kokunero, Email: wanillokokunero@gmail.com]
10:13:45 PM: DEBUG [persona_dao.py:80 ID Persona: 14, Nombre: Paco, Apellido: Pa, Email: pacopa@gmail.com]
10:13:45 PM: DEBUG [persona_dao.py:80 ID Persona: 15, Nombre: Kike, Apellido: Montilla, Email: kikemontilla@gmail.com]
10:13:45 PM: DEBUG [persona_dao.py:80 ID Persona: 17, Nombre: Aitor, Apellido: Menta, Email: aitormenta@gmail.com]
04:26:37 AM: DEBUG [conexion.py:23 Conectado a la BD: <connection object at 0x0000027CC145CE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>]
04:26:37 AM: DEBUG [conexion.py:36 Se abrio el cursor con exito: <cursor object at 0x0000027CC2FB6580; closed: 0>]
04:26:37 AM: INFO [conexion.py:59 <cursor object at 0x0000027CC2FB6580; closed: 0>]
04:26:37 AM: DEBUG [conexion.py:56 Se han cerrado los objetos conexion y cursor]
